.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
 test_**.yaml
+.idea
```

Comment:
Was this added accidentally?

Reply:
Idea and vscode is popular ide, `.idea` is jetbrains idea's project dir, Do I need to remove of it?

Reply:
OK,removed